src/modules/app/actions/__mocks__/update-blockchain.js

```diff
@@ -0,0 +1,8 @@
+const mockUpdateBlockchain = jest.genMockFromModule("augur.js");
+
+mockUpdateBlockchain.updateBlockchain = data => ({
```

Comment:
Why did we need to mock this?

Reply:
"updateBlockChain" was previously rewired in sync-blockchain-test... I could look into using a simpler mocking method tho? since it's just a function, it might be possible.